examples/map/difference_images.py

```diff
@@ -0,0 +1,62 @@
+"""
+===========================
+Plotting a difference image
+===========================
+How to compute and plot a difference image.
+The example uses `sunpy.map.MapSequence` to compute a difference image and then plot it.
+This basic method works for base difference or running difference. Just change whether
+you're subtracting the previous image or the first image in a sequence.
+"""
+###########################################################################
+
+#sphinx_gallery_thumbnail_number = 2
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import astropy.units as u
+from sunpy.net import Fido, attrs as a
+import sunpy.map
+
+###########################################################################
+# First we'll download a couple images and store them in a
+# sunpy.map.MapSequence(). This could be from whatever imager.
+# Here, we use SDO/AIA 304 Å.
+
+instrument = a.Instrument('AIA')
+wave = a.Wavelength(30 * u.nm, 31 * u.nm)
+result = Fido.search(a.Time('2015-06-18T00:00:00', '2015-06-18T00:00:10') |
```

Comment:
I think this query would be a bit more readable written as,

```python
 result = Fido.search(
    a.Time('2015-06-18T00:00:00', '2015-06-18T00:00:10') | a.Time('2015-06-18T01:03:30', '2015-06-18T01:03:35'), 
    instrument,
    wave
)
```

(Sorry for the lack of a suggestion. Apparently GitHub does not like multi-line suggested changes).

Reply:
I did the line break for `instrument` and `wave` (and changed to `wavelength`) but didn't put the two times on a single line since that's > 80 characters and breaks PEP8 (unless we ignore that rule?)